Expo1900/Expo1900/Contorller/KoreaExpositionViewController.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  KoreaExpositionViewController.swift
+//  Expo1900
+//
+//  Created by 김준건 on 2021/07/07.
+//
+
+import UIKit
+
+class KoreaExpositionViewController: UIViewController {
+    @IBOutlet weak var koreaExpositionTableView: UITableView!
+    @IBOutlet weak var koreaExpositionViewTitle: UINavigationItem!
+    private var koreaExpositions: [KoreaExposition] = []
+    private let tableViewCellIdentifer: String = "KoreaExpositionCell"
+    private let viewTitle: String = "한국의 출품작"
+    private let moveToDetailSegueName: String = "MoveToDetail"
+
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        koreaExpositionViewTitle.title = viewTitle
+        decodingKoreaExpositionData()
+        koreaExpositionTableView.dataSource = self
+        koreaExpositionTableView.delegate = self
+    }
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        if segue.identifier == moveToDetailSegueName {
+            if let detailViewController = segue.destination as? KoreaExpositionDetailViewController,
+               let index = sender as? Int {
+                let detailData: KoreaExposition = koreaExpositions[index]
+                detailViewController.koreaExpositionItem = detailData
+            }
+        }
+    }
+    
+    private func decodingKoreaExpositionData() {
+        let decoder: JSONDecoder = JSONDecoder()
+        if let dataAsset: NSDataAsset = NSDataAsset.init(name: "items") {
+            do {
+                koreaExpositions = try decoder.decode([KoreaExposition].self, from: dataAsset.data)
+            } catch {
+                print("Decoding Error")
+            }
+        }
+    }
+}
+
+extension KoreaExpositionViewController: UITableViewDataSource, UITableViewDelegate {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return koreaExpositions.count
+    }
+    
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell: UITableViewCell = tableView.dequeueReusableCell(withIdentifier: tableViewCellIdentifer, for: indexPath)
```

Comment:
```suggestion
guard let cell = tableView.dequeueReusableCell(withIdentifier: tableViewCellIdentifer, for: indexPath) as? KoreaExpositionCustomCell else {
         return UITableViewCell()
     }
```

요렇게 한줄로도 처리가 가능할 것 같아요 :)
> 성능상 차이는 없고 스타일 이기 때문에 참고만해주세요